packages/notifi-react-card/lib/hooks/useSubscriptionCard.ts

```diff
@@ -31,25 +31,30 @@ export const useSubscriptionCard = (
     state: 'loading',
   });
   const { demoPreview } = useNotifiDemoPreviewContext();
-  const { client } = useNotifiClientContext();
+  const {
+    client,
+    canary: { isActive: isCanaryEnabled, frontendClient },
```

Comment:
is the canary something we'd want to expose? wdyt @juni-notifi ?

Reply:
ah i think no harm in exposing it to customers